--- a/AppRun
+++ b/AppRun
@@ -11,16 +11,7 @@
 args=("$@")
 NUMBER_OF_ARGS="$#"
 
-if [ -z "$APPDIR" ] ; then
-  # Find the AppDir. It is the directory that contains AppRun.
-  # This assumes that this script resides inside the AppDir or a subdirectory.
-  # If this script is run inside an AppImage, then the AppImage runtime likely has already set $APPDIR
-  path="$(dirname "$(readlink -f "${THIS}")")"
-  while [[ "$path" != "" && ! -e "$path/$1" ]]; do
-    path=${path%/*}
-  done
-  APPDIR="$path"
-fi
+APPDIR="/opt/ytmdesktop"
 
 export PATH="${APPDIR}:${APPDIR}/usr/sbin:${PATH}"
 export XDG_DATA_DIRS="./share/:/usr/share/gnome:/usr/local/share/:/usr/share/:${XDG_DATA_DIRS}"
@@ -40,9 +31,9 @@
 {
   if [ $isEulaAccepted == 1 ] ; then
     if [ $NUMBER_OF_ARGS -eq 0 ] ; then
-      exec "$BIN"
+      exec "$BIN" --no-sandbox
     else
-      exec "$BIN" "${args[@]}"
+      exec "$BIN" "${args[@]}" --no-sandbox
     fi
   fi
 }
