--- a/src/meson.build
+++ b/src/meson.build
@@ -36,21 +36,3 @@
   'search_provider.rs',
 )

-cargo_release = custom_target(
-  'cargo-build',
-  build_by_default: true,
-  input: sources,
-  output: meson.project_name(),
-  console: true,
-  install: true,
-  install_dir: bindir,
-  depends: resources,
-  command: [
-    cargo_script,
-    meson.build_root(),
-    meson.source_root(),
-    '@OUTPUT@',
-    profile,
-    meson.project_name(),
-  ]
-)
