--- a/usr/share/ganttproject/ganttproject
+++ b/usr/share/ganttproject/ganttproject
@@ -3,6 +3,8 @@
 # This script launches GanttProject. It can be symlinked and can be ran from
 # any working directory

+CUSTOM_JAVA="/opt/liberica-jre-bin-13/"
+
 SCRIPT_FILE="$0"

 find_ganttproject_home() {
@@ -147,6 +149,9 @@
   if [ $USE_BUNDLED_RUNTIME == 1 ]; then
     check_java "$GP_HOME/runtime/bin/java" && return 0;
   fi
+  if [ ! -z "$CUSTOM_JAVA" ]; then
+    check_java "$CUSTOM_JAVA/bin/java" && return 0;
+  fi
   if [ ! -z "$JAVA_HOME" ]; then
     check_java "$JAVA_HOME/bin/java" && return 0;
   fi
